src/plone/api/content.py

```diff
@@ -261,7 +261,10 @@ def copy(source=None, target=None, id=None, safe_id=False):
 
     new_id = copy_info[0]['new_id']
     if id:
-        return rename(obj=target[new_id], new_id=id, safe_id=safe_id)
+        if not safe_id and id in target:
+            raise InvalidParameterError
```

Comment:
How about "Duplicate ID '{id}' in '{target}' for '{source}'"?
ie: `InvalidParameterError: Duplicate ID 'contact' in '<ATFolder at about>' for '<ATDocument at contact>'`